connect/transforms/src/test/java/org/apache/kafka/connect/transforms/ExtractFieldTest.java

```diff
@@ -111,7 +157,24 @@ public void nonExistentFieldWithSchemaShouldFail() {
             xform.apply(record);
             fail("Expected exception wasn't raised");
         } catch (IllegalArgumentException iae) {
-            assertEquals("Unknown field: nonexistent", iae.getMessage());
+            assertEquals("Unknown field: SingleFieldPath{version=V1, path=nonexistent}", iae.getMessage());
```

Comment:
This error message is less human-readable. Could we preserve the existing one? I don't think the syntax version is necessary; we can just use the path as specified by the user in the transform config.